lib/src/component/abstract_transition.dart

```diff
@@ -187,16 +190,19 @@ abstract class AbstractTransitionComponent<T extends AbstractTransitionProps, S
       skipCount = 0;
     }
 
```

Comment:
Hm, I wonder if we should just always do the following here:
```dart
_cancelTransitionEventListener();
_cancelTransitionEndTimer();
```
to clean up any previous calls